config.json

```diff
@@ -752,7 +752,12 @@
       "core": true,
       "difficulty": 6,
       "slug": "bank-account",
-      "topics": null,
+      "topics": [
+        "arrays",
```

Comment:
Why did you add this as a topic? I don't think you need to use arrays to solve this exercise. I know arrays are used in one of the tests, but not in a way that requires the person implementing the exercise to use arrays at all. Does that make sense? So I'm not sure arrays are important enough to this exercise to require it to be added as a topic. What do you think?

Reply:
:+1:

Reply:
I included this topic because it was being used at one the places. But, now just as you mentioned , I agree that arrays don't have anything to do with problem concept , so I'm removing it. Thank you.